markdown/blog/2024-june-summary.md

```diff
@@ -0,0 +1,70 @@
+
+## AsyncAPI Conf on Tour
+We are fast approaching the [Call for Speakers for AACoT'24 London deadline](https://conference.asyncapi.com/venue/london). We encourage anyone to submit their proposals and share your expertise with the community.
+
+
+As we plan for the Online Edition in October, we intend to host the conference on Wednesday, 30th October. Please share your thoughts or vote in the [AsyncAPI Online Conference discussion](https://github.com/orgs/asyncapi/discussions/1297) if you agree or would like to propose a new date.
+
+## AsyncAPI Mentorship Program
+We've had a few people asking if we'll be hosting our annual AsyncAPI Mentorship Program this year, and the answer is yes. We're planning to start the program around September, as we're currently focusing on mentorships under Google Summer of Code. Although this may seem far off, it will allow us time to plan properly, especially as we aim to include more Maintainership projects in the program. As we are in preparation for this, you can read about how the previous [AsyncAPI Mentorship Program in 2023 went](https://www.asyncapi.com/blog/2023-mentorship-summary).
+
+Please keep an eye on our Slack channel for further announcements.
+
+## AsyncAPI Bounty Program
+The  AsyncAPI Bounty Program Q3 is in full swing, with 9 out of 10 issues already assigned and being worked on. Meanwhile, half of the Q2 issues are still in progress. We are pleased to see the program positively impacting the community. 
+We want to express our gratitude to [Viacheslav Turovskyi](https://github.com/aeworxet) for his dedication to running the program smoothly and ensuring that participants adhere to the Bounty Rules. For more details on the submitted issues, please take a look at the [AsyncAPI Bounty Program Board](https://github.com/orgs/asyncapi/projects/36).
+
+## Community Discussions
+
+### Website Redesign
+In recent weeks, [Aishat Muibudeen](https://github.com/Mayaleeeee) (Maya) has been conducting a UX audit on the AsyncAPI website and has identified user experience issues. In the audit, she proposed redesigning the website to address these issues and improve the overall experience. Please read the [Proposal to Redesign the AsyncAPI Website](https://github.com/asyncapi/website/issues/3018) for more details.
+
+### Marketing Working Group
+We have a new Marketing Working Group; the group aims to use marketing strategies to accomplish two key objectives: promoting AsyncAPI adoption and showcasing community achievements. It envisions close collaboration between the AsyncAPI community and sponsors by strategically demonstrating AsyncAPI capabilities and celebrating community successes. If you're interested in participating, please join the [Marketing WG meeting on Tuesday, 2nd July, at 14:00 UTC](https://github.com/asyncapi/community/issues/1292).
```

Comment:
2nd of July is tomorrow, so I guess better to remove the info about meeting, and maybe refer people to dedicated slack channel?